src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -485,12 +524,228 @@ public void createSnapshotRowIdsParquetFile(
     executeSynapseQuery(sqlCreateRowIdTable.render());
   }
 
+  /**
+   * Create Snapshot by Asset First, create parquet file for root table, including only rows that
```

Comment:
I think you missed punctuation here? 
```suggestion
   * Create Snapshot by Asset. First, create parquet file for root table, including only rows that
```